.github/workflows/build.yml

```
@@ -8,17 +8,29 @@ on:
     types:
       - created
   workflow_dispatch:
+    inputs:
+      version:
+        description: "バージョン情報（A.BB.C / A.BB.C-preview.D）"
+        required: true
+      prerelease:
+        description: "プレリリースかどうか"
+        type: boolean
+        default: true
+      code_signing:
+        description: "コード署名する"
+        type: boolean
 
 env:
   VOICEVOX_ENGINE_REPO_URL: "https://github.com/VOICEVOX/voicevox_engine"
   VOICEVOX_ENGINE_VERSION: 0.12.2
   VOICEVOX_RESOURCE_VERSION: 0.12.1
   VOICEVOX_EDITOR_VERSION:
-    |- # releaseのときはタグが、それ以外は999.999.999がバージョン名に
-    ${{ github.event.release.tag_name != '' && github.event.release.tag_name || '999.999.999' }}
+    |- # releaseタグ名か、workflow_dispatchでのバージョン名か、999.999.999が入る
+    ${{ github.event.release.tag_name || github.event.inputs.version || 'latest' }}
```

コメントでは`999.999.999`になっていますが、実際には`latest`が指定されています。どちらが正しいのでしょうか...?

---

あ！999の方が正しいです！ご指摘ありがとうございます！！

```suggestion
    ${{ github.event.release.tag_name || github.event.inputs.version || '999.999.999' }}
```